### PR TITLE
6 design system implementation styled component

### PR DESCRIPTION
**Pull request by jasonzhwang**
Create Design System for the project, make developers easier to refer different color numbers and headings

image

### CODE REVIEW

File: src/components/Design/Colors/Primary/PrimaryColor.tsx [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import styled from 'styled-components';
+
+import Colors from '../../Colors';
+
+const pimaryColor = {
+  'Cyan Dark': '#2B788B',
```
Comment (Prom1s1ngYoung) [inline src/components/Design/Colors/Primary/PrimaryColor.tsx:7]: They are all keys of the 'primaryColor' Object, you should use maybe camel case naming convention for the keys. Except this, the type of keys are already String type, you should remove all single quotes here.
For example: 
cyanDark: '#BBBBBB',

Comment (jasonzhwang) [reply src/components/Design/Colors/Primary/PrimaryColor.tsx]: Hi Young,

"PrimaryColor" keys are updated. Thanks.

File: src/components/Design/design.module.scss [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  display: flex;
+}
+.fontBasestyle {
+  font-family: 'Dela Gothic One';
+  font-style: normal;
+  font-weight: 400;
+  letter-spacing: -0.04em;
+  line-height: 120%;
+  font-size: 32px;
+}
```
Comment (Double-Twelve) [inline src/components/Design/design.module.scss:11]: 1. should be `fontBaseStyle`
2. the default value of `font-style` is `normal` and `font-weight` is `400`, so don't need to be set

File: src/components/Design/design.module.scss [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+.HeadingBasestyle {
+  font-family: Dela Gothic One;
+  font-style: normal;
+  font-weight: 400;
+  letter-spacing: -0.04em;
+  line-height: 120%;
+  color: '#000000';
+}
```
Comment (Double-Twelve) [inline src/components/Design/design.module.scss:20]: 1. should be `HeadingBaseStyle`
2. `font-family` should enclosed it in quotes
3. the default value of `font-style` is `normal` and `font-weight` is `400`, so don't need to be set
4. `color` don't need quotes

File: src/components/Design/Colors/Primary/PrimaryColor.tsx [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+  cyanDark: '#2B788B',
+  cyanLight: '#C3DCE3',
+  pinkDark: '#945069',
+  pinkLight: '#F2D4DC',
+  Black: '#000000',
+  greyLight: '#F6F5F4',
+  greyMedium: '#E0E0E0',
+  greyIcon: '#BABABA',
+  greyDark: '#757575',
+};
```
Comment (Double-Twelve) [inline src/components/Design/Colors/Primary/PrimaryColor.tsx:15]: 1. should be `primaryColor`
2. should be `black`

**Comment by Double-Twelve** [comment]
In addition, in the components folder, the suffix of your index file is `ts`, please change it to `tsx`

File: src/components/Design/design.module.scss (hunk 3 above)
Comment (jasonzhwang) [reply src/components/Design/design.module.scss:20]: Hi Double,

thanks for your detailed explanation. All issues have been fixed. Cheers.

File: src/pages/design.tsx [hunk 5]
```diff
@@ -0,0 +1,6 @@
+import React from 'react';
+import DesignPage from '../components/DesignPage';
+
+export default function design() {
+  return <DesignPage />;
+}
```
Comment (Double-Twelve) [inline src/pages/design.tsx:6]: the function's name to be in PascalCase

**Comment by Double-Twelve** [comment]
In the component folder, there are two folders, naming `Design` and `DesignPage`, so what is the difference between them?

**Comment by jasonzhwang** [comment]
> In the component folder, there are two folders, naming `Design` and `DesignPage`, so what is the difference between them?

I refactored the 'Design' folder to the 'DesignPage', but it somehow keep both 'Design' and 'DesignPage' folders in merge conflict.

**Comment by jasonzhwang** [comment]
> In the component folder, there are two folders, naming `Design` and `DesignPage`, so what is the difference between them?

Hi Double,

Sorry for misunderstanding your idea. The function name has been changed to DesignPage. And also removed redundant folder. Thanks.

**Review by Double-Twelve — APPROVED** [review]

**Pull request merged**